tss-esapi-sys/Cargo.toml

```diff
@@ -14,7 +14,7 @@ links = "tss2-esys"
 rust-version = "1.66.0"
 
 [build-dependencies]
-bindgen = { version = "0.66.1", optional = true }
+bindgen = { version = "0.69.4", optional = true }
```

Comment:
Yes, but only for the versions that Parsec depends on. Since this is for main, and Parsec is still using v7, I think we're good, we can bump the `-sys` version when we release a new v8.

@gowthamsk-arm @tgonzalezorlandoarm - be advised!

Reply:
Thanks for the heads up. We are planning to update bindgen for the rest of the crates (I think we've already started with some of them)